app/tests/spec/lib/fxa-client.js

```diff
@@ -294,12 +294,12 @@ function (chai, $, sinon, FxaClient, p, testHelpers, Session, FxaClientWrapper,
     });
 
     describe('signIn', function () {
-      it('signin with unknown user should call errorback', function () {
+      it('signin with unknown user should fail', function () {
         sinon.stub(realClient, 'signIn', function () {
           return p.reject(AuthErrors.toError('UNKNOWN_ACCOUNT'));
         });
 
-        return client.signIn('unknown@unknown.com', 'password', relier)
+        return client.signIn('unknown@unknown.com', 'password', client.SIGNIN_REASON.SIGN_IN, relier)
```

Comment:
Having to specify SIGNIN_REASON.SIGN_IN here seems unpleasant, you have to type "sign in" three separate times.  Can we shuffle ordering of arguments to make this optional, and default it to SIGN_IN internally?